chore(release): 3.5.0 (#300)
imgix/imgix.js: README.md

**Please note**: `ix-params` must be a valid JSON string. This means that keys and string values must be surrounded by double quotes, e.g., `"fit": "crop"`.

### `ix-sizes` attribute

> Creates an `img` tag with the `sizes` attribute automatically updating to match the image's display size.

Reviewer: This is really nitpicky but the "Creates an `img` tag" might read a little confusing because it's an attribute on an existing `img` tag? Although the same issue exits under the `ix-src` [section](https://github.com/imgix/imgix.js/pull/300/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R97)."

How about:
```suggestion
When set to `auto`, automatically updates an `img` tag's `sizes` attribute to match the image's display size.
```

Reviewer: On a separate note, does "`ix-sizes` only work with `ix-src`/`ix-path`? If so, we should call that out.

  alt="A hot air balloon on a sunny day"
  ix-sizes="auto"
>
```

> **Please note**: the image width has to be calculable before the image has loaded, otherwise `sizes` will not match the width of the displayed image. In most cases, using the CSS rule `img[ix-sizes="auto"] { display: block; width: 100%; }` will ensure the image's `width` is calculable before it has loaded.

Generates HTML similar to the following

``` html
<img
  ix-src="https://assets.imgix.net/unsplash/hotairballoon.jpg?w=300&amp;h=500&amp;fit=crop&amp;crop=right"
  alt="A hot air balloon on a sunny day"
  ix-sizes="auto"
  sizes="200px"
  srcset="
    https://assets.imgix.net/unsplash/hotairballoon.jpg?w=100&amp;h=167&amp;fit=crop&amp;crop=right 100w,
    https://assets.imgix.net/unsplash/hotairballoon.jpg?w=200&amp;h=333&amp;fit=crop&amp;crop=right 200w,
    …
    https://assets.imgix.net/unsplash/hotairballoon.jpg?w=2560&amp;h=4267&amp;fit=crop&amp;crop=right 2560w
  "
  src="https://assets.imgix.net/unsplash/hotairballoon.jpg?w=300&amp;h=500&amp;fit=crop&amp;crop=right"
  ix-initialized="ix-initialized"
>
```

> When using `ix-sizes="auto"`, the browser will not have the `sizes` attribute to reference on first render but only after `imgix,js` has loaded. This is why it's recommended to manually set `sizes` whenever possible.

Reviewer: ```suggestion
When using `ix-sizes="auto"`, the browser will not have the `sizes` attribute to reference on first render but only after `imgix.js` has loaded. This is why it's recommended to manually set `sizes` whenever possible.
```

Author: Thanks, will fix and force push to keep commit history clean.
